### PR TITLE
Update CHANGELOG - 3.4, 3.5 - fix PRISMA-2023-0056

### PR DESCRIPTION
**Pull request by vivekpatani**
- Address [PRISMA-2023-0056](https://github.com/etcd-io/etcd/issues/17268#issuecomment-2050252647) changes.

### CODE REVIEW

**Comment by vivekpatani** [comment]
@ivanvc @ahrtr PTAL, the PR I've been meaning together for a couple of weeks :D

**Comment by ivanvc** [comment]
Thanks @vivekpatani :bow: 

/ok-to-test

File: CHANGELOG/CHANGELOG-3.4.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 - Upgrade [bbolt to 1.3.9](https://github.com/etcd-io/etcd/pull/17484).
 - Compile binaries using [go 1.21.8](https://github.com/etcd-io/etcd/pull/17538).
 - Upgrade [google.golang.org/protobuf to v1.33.0 to address CVE-2024-24786](https://github.com/etcd-io/etcd/pull/17554).
+- Fix [PRISMA-2023-0056](https://github.com/etcd-io/etcd/pull/17580).
```
Comment (ivanvc) [inline CHANGELOG/CHANGELOG-3.4.md:38]: Suggestion to specify the dependency bumped.

```suggestion
- Upgrade github.com/sirupsen/logrus to v1.9.3 to address [PRISMA-2023-0056](https://github.com/etcd-io/etcd/pull/17580).
```

**Review by ivanvc — COMMENTED** [review]
I left a suggestion for 3.4 and 3.5. I haven't seen that level of detail at 3.6 dependencies.

File: CHANGELOG/CHANGELOG-3.4.md (hunk 1 above)
Comment (vivekpatani) [reply CHANGELOG/CHANGELOG-3.4.md]: Update v3.4/3.5 @ivanvc thanks.

File: CHANGELOG/CHANGELOG-3.6.md [hunk 2]
```diff
@@ -100,5 +100,6 @@ See [List of metrics](https://etcd.io/docs/latest/metrics/) for all metrics per
 
 - Use Distroless as base image to make the image less vulnerable and reduce image size.
 - [Upgrade grpc-gateway from v1 to v2](https://github.com/etcd-io/etcd/pull/16595).
+- Fix [PRISMA-2023-0056](https://github.com/etcd-io/etcd/pull/17445).
```
Comment (ahrtr) [inline CHANGELOG/CHANGELOG-3.6.md:103]: FYI. https://github.com/etcd-io/etcd/tree/main/CHANGELOG#change-log-rules
```suggestion
```

**Comment by ivanvc** [comment]
/retitle Update CHANGELOG - 3.4, 3.5 - fix PRISMA-2023-0056

**Review by serathius — APPROVED** [review]

**Review by ahrtr — APPROVED** [review]

**Pull request merged**